travis-ci/terraform-config
# Downgrade cs50 instance type

## What is the problem that this PR is trying to fix?
Fixes https://github.com/travis-pro/team-blue/issues/812

## What approach did you choose and why?
Change the instance type, per @acnagy's suggestion (didn't check the calculation since I have no info about cs50's workload).

## How can you test this?
`make clean plan` output:

```An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place
-/+ destroy and then create replacement

Terraform will perform the following actions:

-/+ module.aws_asg_com.null_resource.cyclist (new resource required)
      id:                          "5058570438494204536" => <computed> (forces new resource)
      triggers.%:                  "4" => "4"
      triggers.config_signature:   "f2388ad6e471dbc785bfec56957a154e19b5201b4fd8edab30d27bab811420a0" => "f2388ad6e471dbc785bfec56957a154e19b5201b4fd8edab30d27bab811420a0"
      triggers.heroku_id:          "cyclist-production-2-com" => "cyclist-production-2-com"
      triggers.ps_scale:           "web=1:Standard-1X" => "web=1:Standard-1X"
      triggers.version:            "v0.4.0" => "v0.5.0" (forces new resource)

  ~ module.aws_asg_cs50.aws_autoscaling_group.workers
      launch_configuration:        "cs50-production-2-workers-com-00612f97ee270c34e4772adf38" => "${aws_launch_configuration.workers.name}"

-/+ module.aws_asg_cs50.aws_launch_configuration.workers (new resource required)
      id:                          "cs50-production-2-workers-com-00612f97ee270c34e4772adf38" => <computed> (forces new resource)
      associate_public_ip_address: "false" => "false"
      ebs_block_device.#:          "0" => <computed>
      ebs_optimized:               "false" => <computed>
      enable_monitoring:           "true" => "true"
      image_id:                    "ami-3889d642" => "ami-3889d642"
      instance_type:               "c3.2xlarge" => "c3.xlarge" (forces new resource)
      key_name:                    "" => <computed>
      name:                        "cs50-production-2-workers-com-00612f97ee270c34e4772adf38" => <computed>
      name_prefix:                 "cs50-production-2-workers-com-" => "cs50-production-2-workers-com-"
      root_block_device.#:         "0" => <computed>
      security_groups.#:           "4" => "4"
      security_groups.1507295911:  "sg-9f2860e5" => "sg-9f2860e5"
      security_groups.15829228:    "sg-0a33b17f" => "sg-0a33b17f"
      security_groups.1625441574:  "sg-e130b294" => "sg-e130b294"
      security_groups.4161445841:  "sg-962860ec" => "sg-962860ec"
      user_data:                   "658853bf7fc8fd9646d1372bf14295ac5ce0ce93" => "658853bf7fc8fd9646d1372bf14295ac5ce0ce93"

-/+ module.aws_asg_cs50.null_resource.cyclist (new resource required)
      id:                          "7150019032147566333" => <computed> (forces new resource)
      triggers.%:                  "4" => "4"
      triggers.config_signature:   "7db8111c7451c5e6f5de8da93229f6f33052ebc6dc467c0a47da4eb303585843" => "7db8111c7451c5e6f5de8da93229f6f33052ebc6dc467c0a47da4eb303585843"
      triggers.heroku_id:          "cyclist-cs50-production-2-com" => "cyclist-cs50-production-2-com"
      triggers.ps_scale:           "web=1:Standard-1X" => "web=1:Standard-1X"
      triggers.version:            "v0.4.0" => "v0.5.0" (forces new resource)

-/+ module.aws_asg_org.null_resource.cyclist (new resource required)
      id:                          "3689826903420961" => <computed> (forces new resource)
      triggers.%:                  "4" => "4"
      triggers.config_signature:   "7ff03c3a171c7159c61f722ea5e8203233b6e29b9f2bbda4d1dc6fff56a2a090" => "7ff03c3a171c7159c61f722ea5e8203233b6e29b9f2bbda4d1dc6fff56a2a090"
      triggers.heroku_id:          "cyclist-production-2-org" => "cyclist-production-2-org"
      triggers.ps_scale:           "web=1:Standard-1X" => "web=1:Standard-1X"
      triggers.version:            "v0.4.0" => "v0.5.0" (forces new resource)


Plan: 4 to add, 1 to change, 4 to destroy.
```

I assume the cyclist changes are due to https://github.com/travis-ci/terraform-config/pull/321 not being applied in prod. 
The cs50 changes look as expected.
